### DISCUSSION TITLE
v2.1.9

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3311] [分享 / Share]
✨**正式版** `v2.1.9`
- `删除广告` 支持屏蔽视频页的创作推广和动态首页的新广告. (#3282, #3269)
- `动态反折叠` 和 `动态过滤器` 的开播记录类型支持新版动态首页. (#3191)
- 排除在直播开放平台的运行 (open-live.bilibili.com). (#3259)
- 检测到由 GreasyFork 安装时禁用反馈入口. (#3277)
- `网址参数清理` 不再在 iframe 中运行. (#2953)
- 修复标题格式化的两个问题. (#3251, #3309, PR 3310 by [cicur](https://github.com/cicur))

----

✨**预览版** `v2.1.9-preview`

- 设置面板移动了搜索框的位置, 添加了检查更新和卸载组件的快捷按钮. (PR #3279 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
- `自定义顶栏` 支持硬核 LV6 会员的图标显示. (#3203)
- `动态过滤器` 支持屏蔽发送动态的面板. (#2447)
- 新增插件 `下载视频 - 手动输入`, 可以手动输入 av / BV 号来进行下载. (#3227)
- Toast 消息能够显示关闭时间的倒计时进度, 且鼠标进入时停止倒计时. (#3204)

☕开发者相关
- 所有 `substr` 调用替换为 `substring`.
- 在 ScopedConsole 的名称间添加了空格, 以改善复制为文本后的可读性.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.9'>v2.1.9</a>.</em>

**Comment by xfht** [comment]
动态页面的夜间模式没修复呢

**Reply by the1812** [reply]
嗯, 应该就剩这个没修复了吧

**Comment by Geeyun-JY3** [comment]
多选后卸载组件建议加个确认对话框，这算是危险行为了。

**Reply by Geeyun-JY3** [reply]
不对，按现在卸载按钮和更新按钮位置的这个设计，还是只要点击卸载按钮就弹确认对话框吧，很难说会不会误触。

**Comment by alj20080701** [comment]
尽快修复一下夜间模式吧，现在这个样子开了夜间模式看着属实有点点难受的感觉

**Comment by adfnkjasd** [comment]
硬核会员还是没能显示呢
